src/hackingBuddyGPT/utils/openai/openai_llm.py

```diff
@@ -36,9 +40,22 @@ def get_response(self, prompt, *, retry: int = 0, **kwargs) -> LLMResult:
         headers = {"Authorization": f"Bearer {self.api_key}"}
         data = {'model': self.model, 'messages': [{'role': 'user', 'content': prompt}]}
 
+        # Log the request payload
+        #
+        # Uncomment the following to log debug output
+        # logging.debug(f"Request payload: {data}")
```

Comment:
could you make this configurable (a `debug-requests` flag instead of commenting logging in and out)?

Reply:
I removed the commented lines as they were temporary.

Not keen to introduce a `debug-requests` flag to add to the complexity of logging via https://github.com/ipa-lab/hackingBuddyGPT/issues/75 and https://github.com/ipa-lab/hackingBuddyGPT/pull/79